microsoft-edge/devtools-guide-chromium/whats-new/2020/08/devtools.md

```diff
@@ -115,7 +115,7 @@ This feature makes it easier to test the way screen reader and keyboard users ex
 
 To turn on this experimental feature, see [Turning an experiment on or off](../../../experimental-features/index.md#turning-an-experiment-on-or-off) and select the checkbox next to **Enable Source Order Viewer**.
```

Comment:
@captainbrosset - per email, there are about 20 tautology sentences that dup the label text of the checkbox.  Merge PR as-is, or clarify those sentences/descriptions in those 20 h2 sections?

Reply:
Yeah I don't think we need to have all of them before we merge this PR. Also, some of the experiment names are pretty self-explanatory like "Enable automatic contrast issue reporting via the Issues Panel" and don't really need any further description.

That said, here are a couple that you might want to add in:

* Show CSP Violations view
  
   "Adds the **CSP Violations** tool which can be used to view all Content Security Policy (CSP) violations detected on the inspected web page."

   Also, please remove the `See [Content Security Policy (CSP)](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/store-policies/csp).` link that's there now. It links to add-ons docs, which are unrelated to devtools.

* Enable full accessibility tree view in the Elements panel

   "Adds a button to the **Elements** panel which can be used to toggle between the DOM tree and the accessibility tree".